.github/workflows/s01-02-availability.yml

```
@@ -0,0 +1,93 @@
+    steps:
+      - uses: actions/checkout@v4
+        with: {ref: 57252484f834ce536644909d46a8c30480041f4a}
+      - uses: pnpm/action-setup@v4
+        with: {version: 10.26.1, run_install: false}
+      - uses: actions/setup-node@v4
+        with: {node-version: 22, cache: pnpm}
+      - run: pnpm install --frozen-lockfile
+      - name: Provision baseline
+        env: {DATABASE_URL: postgresql://postgres:postgres@localhost:5432/mustashark_s01_02_migration}
+        run: pnpm --filter @workspace/db push
+      - uses: actions/checkout@v4
```

actionlint (1.7.12)

[error] could not parse as YAML: did not find expected ',' or '}'

(syntax-check)